pkg/deploy/registry/deploylog/rest_test.go

```diff
@@ -144,15 +146,49 @@ func mockREST(version, desired int64, status api.DeploymentStatus) *REST {
 				Phase: kapi.PodRunning,
 			},
 		}
-		fakePn.PrependReactor("get", "pods", func(action ktestclient.Action) (handled bool, ret runtime.Object, err error) {
+		oldPn.PrependReactor("get", "pods", func(action ktestclient.Action) (handled bool, ret runtime.Object, err error) {
```

Comment:
rolling and recreate invoke commands from `k8s.io/kubernetes/pkg/kubectl` package, which is using clientset only in 1.5, which we'll get after the rebase. Example line:
`kubectl.ScalerFor(kapi.Kind("ReplicationController"), oldClient)` for that reason we need old k8s client in only certain places, which will be removed after the rebase. See my discussion with Michalis in this PR.